WakaTime.sublime-settings

```diff
@@ -3,10 +3,6 @@
 // This settings file will be overwritten when upgrading.
 
 {
-    // Your api key from https://wakatime.com/#apikey
```

Comment:
We need these default settings, since some people configure their api key in Sublime without wakatime.cfg.

Reply:
added back